app/src/main/java/protect/card_locker/BarcodeValues.java

```diff
@@ -1,11 +1,14 @@
 package protect.card_locker;
 
+import androidx.annotation.Nullable;
+
 public class BarcodeValues {
+    @Nullable
     private final String mFormat;
     private final String mContent;
     private String mNote;
 
-    public BarcodeValues(String format, String content) {
+    public BarcodeValues(@Nullable String format, String content) {
         mFormat = format;
         mContent = content;
     }
```

Comment:
Do we want to ensure we only accept sane values here? Perhaps don't use String as input type but CatimaBarcode?

Reply:
Made it so this BarcodeValues must be a CatimaBarcode format or null